csvcubed/csvcubed/cli/inspect/inspect.py

```diff
@@ -35,8 +36,9 @@ def inspect(csvw_metadata_json_path: Path) -> None:
     metadata_processor = MetadataProcessor(csvw_metadata_json_path)
     csvw_metadata_rdf_graph = metadata_processor.load_json_ld_to_rdflib_graph()
 
-    assert csvw_metadata_rdf_graph is not None
-
+    if csvw_metadata_rdf_graph is None:
+        raise RDFGraphCannotBeNoneException
```

Comment:
Don't you need the open and closing brackets here? Will it call the `__init__` function?

Reply:
@robons Sorry, rind-replace error - the `()` should be there. I will sort.